src/test/java/com/wikia/webdriver/testcases/adstests/mobileadstests/TestDfpParamsPresentMobile.java

```diff
@@ -2,6 +2,7 @@
 
 import com.wikia.webdriver.common.dataprovider.mobile.MobileAdsDataProvider;
 import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
```

Comment:
I recommend you install save actions, so that your imports get removed automatically

You can add save actions to intelliJ Idea in order to:
- organize imports (remove the unused ones)
- reformat code
basically to save time

The exact steps are described in this stacoverlow thread 
http://stackoverflow.com/questions/3707316/save-actions-in-intellij-idea/32137623#32137623